app/controllers/concerns/workflow/controller.rb

```diff
@@ -86,6 +84,19 @@ def send_transition_email
       end
     end
 
+    # Who should receive emails about changes to a `project`s workflow position.
+    # In lieu of a subscription based model where users can manage their own desired status updates.
+    # NOTE: yuk.
+    def notifiable_users
```

Comment:
So I guess ODR user only needs an email to inform them they need to action something and do not care about status updates. And then we cycle back to application managers being pseudo users.

Reply:
Sorry, I'm not sure I follow you on this one? Care to elaborate?

Reply:
Sorry. I think my point is on the other comment around our ODR user group size of one

Reply:
👍